alibi_detect/cd/base.py

```diff
@@ -240,7 +240,7 @@ def test_probs(
         return p_val, dist
 
     @abstractmethod
-    def score(self, x: Union[np.ndarray, list]) -> Tuple[float, float, np.ndarray, np.ndarray]:
+    def score(self, x: Union[np.ndarray, list]) -> Tuple[float, float, np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
```

Comment:
Note for future - reconsider making `score` a private method. This would also allow us to use more maintainable data structures for the return type, e.g. a dataclass.

Reply:
Opened #666 for this